examples/vision/oxford_pets_image_segmentation.py

```diff
@@ -161,7 +161,8 @@ def get_model(img_size, num_classes):
         previous_block_activation = x  # Set aside next residual
 
     # Add a per-pixel classification layer
-    outputs = layers.Conv2D(num_classes, 3, activation="sigmoid", padding="same")(x)
+    _activation = "softmax" if num_classes > 1 else "sigmoid"
```

Comment:
Nit: please don't use a `_` prefix -- just `activation`.

Reply:
I chose just to change the activation function to softmax in the end, as in this example you're using num_classes greater than 1